### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**

### CODE REVIEW

**Comment by joaoantoniocardoso** [comment]
I can't know for sure, but I'm assuming all the remaining warnings are about dead code that are going to be used in near future.

**Comment by patrickelectric** [comment]
> I can't know for sure, but I'm assuming all the remaining warnings are about dead code that are going to be used in near future.

Since mostly are not used right now, you can add an underscore in the beggning of the name. 
https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#explanation-13

File: rustfmt.toml [hunk 1]
```diff
@@ -1 +1 @@
-edition = "2018"
+edition = "2021"
```
Comment (patrickelectric) [inline rustfmt.toml:1]: Can you squash with 99855dc ?

File: Cargo.lock [hunk 2]
```diff
@@ -13,7 +13,7 @@ dependencies = [
  "futures-core",
  "futures-sink",
  "log 0.4.14",
- "pin-project 0.4.28",
+ "pin-project 0.4.29",
```
Comment (patrickelectric) [inline Cargo.lock:16]: Why this PR is changing the lock file ? This should not happen.

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: Cargo.lock (hunk 2 above)
Comment (joaoantoniocardoso) [reply Cargo.lock:16]: oops, I think this had occurred with the `cargo update`.

File: rustfmt.toml (hunk 1 above)
Comment (joaoantoniocardoso) [reply rustfmt.toml:1]: yep. I'll change it.

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**